src/main/mondrian/rolap/RolapAggregator.java

```diff
@@ -270,7 +269,7 @@ public boolean supportsFastAggregates(
      * These aggregators are NOT singletons like the above aggregators; rather,
      * each is different because of the fact count column expression.
      */
-    protected static abstract class BaseAggor extends RolapAggregator {
+    public static abstract class BaseAggor extends RolapAggregator {
```

Comment:
This is not necessary. Move your test in the right package and you won't need to make it public. We don't make things public just to test them like this.